libtelemetry/src/full/java/com/mapbox/android/telemetry/crash/CrashReporterJobIntentService.java

```diff
@@ -1,71 +0,0 @@
-package com.mapbox.android.telemetry.crash;
-
-import android.content.Context;
-import android.content.Intent;
-import androidx.annotation.NonNull;
-import androidx.annotation.VisibleForTesting;
-import androidx.core.app.JobIntentService;
-import android.util.Log;
-import com.mapbox.android.core.FileUtils;
-import com.mapbox.android.telemetry.CrashEvent;
-
-import java.io.File;
-
-import static com.mapbox.android.telemetry.MapboxTelemetryConstants.MAPBOX_TELEMETRY_PACKAGE;
-
-/**
- * This is a background job that sends crash events to the telemetry endpoint
- * at startup.
- */
-public final class CrashReporterJobIntentService extends JobIntentService {
```

Comment:
Noting that this is technically breaking SemVer as it's a `public` class which is not under `internal`

Reply:
@nkukday though they arrived after merging the PR - did you have a chance to address @Guardiola31337's comments, maybe as tail work?